bench.py

```diff
@@ -564,7 +648,9 @@ def print_output(library, algorithm, stages, params, functions,
                         result['algorithm_parameters']['init'] = 'random'
                 result['algorithm_parameters'].pop('handle', None)
             output.append(result)
+        print('# Intel(R) Extension for Scikit-learn case result:')
         print(json.dumps(output, indent=4))
+        print('# Intel(R) Extension for Scikit-learn case finished.')
```

Comment:
sklbench may not use sklearnex, so, this outputs will be confusing for some cases